src/pages/FindYourCommunity/YourLocalArea/FindSchool/FindSchool.tsx

```diff
@@ -67,12 +65,23 @@ const FindSchool: FC = () => {
       <BackButton theme="blue" />
       <Card className={styles.subContainer}>
         <h1>Find a school near {state.postcode.toUpperCase()}</h1>
+
+        <div className={styles.subText}>
+          <p>
+            If the school you are looking for has not joined Donate to Educate yet,{' '}
+            <Link to={Paths.LOCAL_CHARITIES} state={{ postcode: state.postcode }}>
+              you can also find nearby charities who may be able to help.
+            </Link>
+          </p>
+        </div>
+
         <ProductsTable
           tableData={schoolData}
           type={InstitutionType.SCHOOL}
           iconColour="#97C8EB"
           productsColumnHeader="Product types available"
           postcode={state.postcode}
+          hideNotJoined
```

Comment:
@JakeReadman thanks for implementing this prop in the `<ProductsTable/>` component, came in very useful here!